src/openms/source/FORMAT/HANDLERS/MzIdentMLDOMHandler.cpp

```diff
@@ -1970,6 +1970,14 @@ namespace OpenMS::Internal
       try
       {
         rank = StringManager::convert(spectrumIdentificationItemElement->getAttribute(CONST_XMLCH("rank"))).toInt();
+        if (rank == 0) 
+        {
+          // MzIdentML ranks are typically 1-based. For some special data (PMF) it can be 0. 
```

Comment:
What the heck? 😂 Do we still need mzidentml?

Reply:
I think MSGFPlus uses it? not sure though...